.vib/magento/cypress/cypress/integration/magento_spec.js

```diff
@@ -0,0 +1,103 @@
+/// <reference types="cypress" />
+import { random, allowDataUsage } from './utils';
+
+before(() => {
+  cy.login();
+  allowDataUsage();
+});
+
+it('allows admin to add a product to the store', () => {
+  cy.get('#menu-magento-catalog-catalog').click();
+  cy.get('.item-catalog-products').click();
+  cy.contains('Salable Quantity');
+  cy.get('#add_new_product-button').click();
+  cy.fixture('products.json').then((product) => {
```

Comment:
Minor comment. Has being any changes to the `fixture` function? I think we don't add the file extension in any of our tests.